openapi/paths/storefront/kyc-liveness-sessions@{id}.yaml

```
@@ -0,0 +1,24 @@
+parameters:
+  - $ref: ../../components/parameters/resourceId.yaml
+get:
+  x-products:
+    - Storefront
+  tags:
+    - KYC documents
+  summary: Retrieve a KYC liveness session
+  operationId: StorefrontGetKycLivenessSession
+  x-sdk-operation-name: get
+  description: Retrieves a KYC liveness session with a specified ID.
```

```suggestion
  description: Retrieves a KYC liveness session with a specified ID.
  security:
    - CustomerJWT: []
```

Seems custom jwt is missing here